src/Components/RedesignHeroPanel.jsx

```diff
@@ -0,0 +1,40 @@
+import { Box, Container, makeStyles, Typography } from "@material-ui/core";
+
+const useStyles = makeStyles((theme) => ({
+  hero: {
+    paddingTop: theme.spacing(16),
+    paddingBottom: theme.spacing(16),
+    backgroundColor: theme.palette.primary.dark,
+    color: theme.palette.primary.contrastText,
+  },
+  heroTitle: {
+    paddingTop: theme.spacing(3),
+    paddingBottom: theme.spacing(3),
+  },
+  heroSubtitle: {
+    maxWidth: 502,
```

Comment:
Can this be changed to ems or rems instead of px? I usually just use px for very small values only.

Reply:
Great Idea Ill update it!